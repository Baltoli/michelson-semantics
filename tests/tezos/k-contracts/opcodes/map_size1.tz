$contract { parameter (map string nat);
storage nat;
code {CAR; SIZE; NIL operation; PAIR}
 } ;
$param { Elt "a" 1 } ;
$storage 111 ;
$ExpectedOutput 1 ;

