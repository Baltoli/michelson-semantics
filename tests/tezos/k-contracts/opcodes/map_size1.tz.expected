<generatedTop>
  <k>
    .
  </k>
  <stack>
    Pair .List 1
  </stack>
  <paramtype>
    map .AnnotationList string .AnnotationList nat .AnnotationList
  </paramtype>
  <storagetype>
    nat .AnnotationList
  </storagetype>
  <balance>
    #Mutez ( 0 )
  </balance>
  <amount>
    #Mutez ( 0 )
  </amount>
  <now>
    #Timestamp ( -1 )
  </now>
  <myaddr>
    #Address ( "InvalidMyAddr" )
  </myaddr>
  <knownaddrs>
    .Map
  </knownaddrs>
  <sourceaddr>
    #Address ( "InvalidSourceAddr" )
  </sourceaddr>
  <senderaddr>
    #Address ( "InvalidSenderAddr" )
  </senderaddr>
  <chainid>
    #ChainId ( 0 )
  </chainid>
</generatedTop>
